README.md

```diff
@@ -418,6 +418,12 @@ vault {
   # before being used. Please see Vault's cubbyhole response wrapping
   # documentation for more information.
   unwrap_token = true
+  
+  # The default lease duration Consul Template will use on a Vault secret that 
+  # does not have a lease duration. This is used to calculate the sleep duration
+  # for rechecking a Vault secret value. This field is optional and will default to
+  # 5 minutes.
+  # default_lease_duration = "60s"
```

Comment:
This documentation has moved while this has been waiting (sorry). If you have a chance and wouldn't mind @nritholtz, please consider updating this PR with the documentation moved into `docs/configuration.md`. If you don't have time just say so, I can make the update if necessary.

Thanks.

Reply:
Ok, will do. I'll try to preserve attribution. Thanks.